scripts/README.md

```diff
@@ -0,0 +1,21 @@
+# Script Summary
```

Comment:
I would consider including this information in the docstring of the new script - as a separate text file it is more likely to be forgotten about over time.